server/middleware/validation.js

```diff
@@ -155,6 +187,10 @@ const validators = {
     }
 };
 
+const getDay = (date) => date.split('-')[2];
```

Comment:
Are there any scenarios wherein there aren't enough dashes in the date string?

Reply:
In the current usage no, the date string comes through as undefined if not all are present so it doesn't reach this stage, however in the interest of reusable code I have updated these methods to check length